modules/channel/src/main/scala/higherkindness/mu/rpc/channel/ManagedChannelConfig.scala

```diff
@@ -32,7 +32,11 @@ final case class OverrideAuthority(authority: String)             extends Manage
 final case class UsePlaintext()                                   extends ManagedChannelConfig
 final case class NameResolverFactory(resolverFactory: NameResolver.Factory)
     extends ManagedChannelConfig
+@deprecated(
+  "This method disables service-config-based policy selection, and may cause problems. See DefaultLoadBalancingPolicy",
+  "0.16.0")
```

Comment:
I guess this is being deprecated in the next release, potentially for `0.17.1` or `0.18.0`?

Reply:
You're right. Good catch